app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
```

Comment:
Yes, you're absolutely right.  Thanks for thinking of it!  

I'd like to do that as a a separate PR.  But I can also see the value of including it here.

What's your preference, @patmbolger ?